node/plain/functionApp/Application/Function1/sample.dat

```diff
@@ -0,0 +1,3 @@
+{
+    "name": "Azure"
```

Comment:
why is this file required?

Reply:
It's generated by VSCode, I was not sure if this file should be removed.. Function works perfectly without it, so I'll remove it for now

Reply:
lets leave it in that case